sklearn/dummy.py

```diff
@@ -410,11 +410,11 @@ class DummyRegressor(MultiOutputMixin, RegressorMixin, BaseEstimator):
         * "constant": always predicts a constant value that is provided by
           the user.
 
-    constant : int or float or array-like of shape (n_outputs,)
+    constant : int or float or array-like of shape (n_outputs,), default=None
```

Comment:
```suggestion
    constant : int, float or array-like of shape (n_outputs,), default=None
```

Reply:
`int or float or...` is fine IMO.

if we're going to write commas, we need to put them everywhere so as not to imply artidicial structure:

`int, float, or array-like...`